### ISSUE TITLE
Start Game 

### ISSUE THREAD

**Issue by YorickLB** [Dylan-Zheng/Rise-of-Kingdoms-Bot#90]
Hello I have a Proplem when the game restarts he does not wait until the game is started properly and thus aborts the game and tries to start it again.

this goes on and on until I stop the bot

does anyone have a solution?
Or can I add in the Task.py a sleep at get_curr_gui_name(self)

**Comment by SillyeGergo** [comment]
Hi!

Does your emulator runs at 1280x720?

**Comment by YorickLB** [comment]
Hi!

Yes it runs on 1280 x 720 

When Bluestacks starts Rok then it goes to 70% and then it goes back to the Bluestacks overview and tries to re-strar Rok again it does that until I cancel it.

otherwise the bot runs great
have only a 2captcha a prblem that he does not do that

**Comment by frogedr** [comment]
@YorickLB Hi have the same issue where do you add the sleep on Task.py and whats the value? If you have discord just add me for give me some help FrogeDR#3343

**Comment by YorickLB** [comment]
Hello I added this to the file Task.py after line 138 time.sleep(60) so it waits after it has started ROK

**Comment by YorickLB** [comment]
you can also look here because I have already changed the file https://github.com/YorickLB/Rise-of-Kingdoms-Bot/tree/main/tasks

**Comment by frogedr** [comment]
Yeah that worked thx but you have some way too contact you i have some errors if can take a look

**Comment by TuanVN008** [comment]
> you can also look here because I have already changed the file https://github.com/YorickLB/Rise-of-Kingdoms-Bot/tree/main/tasks

I have an error solving the captcha, help me pls
